features/endpoints/grafana/client/pom.xml

```diff
@@ -21,7 +21,7 @@
             <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
             <Bundle-Version>${project.version}</Bundle-Version>
-            <Embed-Dependency>*;artifactId=okhttp|gson|okio</Embed-Dependency>
+            <!--<Embed-Dependency>*;artifactId=okhttp|gson|okio</Embed-Dependency>-->
```

Comment:
It's important to understand why we are doing this. Why did this fail when we upgraded okhttp3.

Reply:
I have generate locally the runtime dependencies in target and okhttp3 was there .